src/metabase/api/user.clj

```diff
@@ -160,9 +160,30 @@
   [clauses]
   (dissoc clauses :order-by :limit :offset))
 
+(defn- group-ids-for-manager
+  "Given a `user-id` return a list of group-ids of which the user is a group manager."
+  [user-id]
+  (map :group_id
+       (t2/query
```

Comment:
nit: slightly nicer to use `t2/select-fn-set` here, with `:group_id` as the function, to return a set of group IDs directly